Test/Integration/WebTestCase.php

```diff
@@ -0,0 +1,55 @@
+<?php declare(strict_types=1);
+
+namespace Test\Integration;
+
+use function App\Utility\setupEnv;
+use GuzzleHttp\Client;
+use Psr\Http\Message\ResponseInterface;
+use Slim\App;
+use PHPUnit\Framework\TestCase;
+
+class WebTestCase extends TestCase
+{
+    /** @var App $app  */
```

Comment:
Personally I don't like this one liners, I'd prefer:

```
/**
 * @var App $app
 */
```

Reply:
I'm of the same preference tbh